# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/IDBCursor.h
# Commit: 9e7a90dcc429
# Full Hash: 9e7a90dcc429194fbe08e63aeed311ce100114cc
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-10-02 03:38:52
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Use const where easily possible. r=ttung,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43631
# ==============================================================================

diff -r e493a6b5f125 -r 9e7a90dcc429 dom/indexedDB/IDBCursor.h
--- a/dom/indexedDB/IDBCursor.h	Tue Oct 01 08:56:37 2019 +0000
+++ b/dom/indexedDB/IDBCursor.h	Tue Oct 01 09:14:04 2019 +0000
@@ -66,7 +66,7 @@
   RefPtr<IDBIndex> mSourceIndex;
 
   // mSourceObjectStore or mSourceIndex will hold this alive.
-  IDBTransaction* mTransaction;
+  IDBTransaction* const mTransaction;
 
   // These are cycle-collected!
   JS::Heap<JS::Value> mCachedKey;